data/categories.txt

```diff
@@ -785,7 +785,7 @@ shop-car
 en:car shop|U+1F697|U+1F698|U+1F699
 ru:автосалон|магазин
 uk:автосалон|магазин
-de:Auto-Shop|Laden
+de:Autohaus
```

Comment:
Thanks. we’ll discuss it with our team. For me, small retail businesses are also ok :)

> On Feb 1, 2016, at 19:39, malenki <notifications@github.com> wrote:
> 
> In data/categories.txt <https://github.com/mapsme/omim/pull/1735#discussion_r51442601>:
> 
> > @@ -785,7 +785,7 @@ shop-car
> >  en:car shop|U+1F697|U+1F698|U+1F699
> >  ru:автосалон|магазин
> >  uk:автосалон|магазин
> > -de:Auto-Shop|Laden
> > +de:Autohaus
> Alex Zolotarev schrieb
> Is "Geschäft" better as a generic "shop" or "shops" than "Laden"? What is the difference?
> "Laden" is usually used for smaller retail businesses, rarely (if all) for supermarkets/discounters and I'd doubt that people in Germany use it at all for car dealers. To describe "Geschäft" quite vaguely: A place where you get $stuff (be it a cleaning for your tuxedo, a car or a new axe) for money
> —
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/1735/files#r51442601>.
>